frontend/src/utils/account-send.ts

```diff
@@ -0,0 +1,144 @@
+import { isAddress, keccak256, toUtf8Bytes, BigNumber, hexZeroPad } from 'src/utils/ethers';
+import localforage from 'localforage';
+import { toAddress, lookupAddress } from 'src/utils/address';
+import { LOCALFORAGE_ACOUNT_SEND_KEY } from 'src/utils/constants';
+import { Web3Provider } from 'src/utils/ethers';
+
+type AccountData = {
+  address: string;
+  advancedMode: boolean;
+  checkbox: boolean;
+};
+
+type KeyData = {
+  count: number;
+  viewingKey: string;
+};
+
+type EncryptedData = {
+  encryptedAddress: string;
+};
+
+type AccountSendData = {
+  amount: string;
+  tokenAddress: string;
+  dateSent: Date;
+  hash: string;
+  userAddress: string;
+  recipientAddress: string;
+  checkbox?: boolean;
+  advancedMode?: boolean;
+};
+
+type DecryptedSendData = {
+  recipientId: string;
+};
```

Comment:
`DecryptedSendData` sounds like it should contain a bunch of data, but it's just a recipient ID. Maybe we should have a better name? Same for `EncryptedData` which is a vague name